packages/i18next/src/lib/functions.ts

```diff
@@ -1,6 +1,6 @@
 import { container } from '@sapphire/pieces';
 import { isObject, NonNullObject } from '@sapphire/utilities';
-import { Guild, Message } from 'discord.js';
+import { Guild, Message, CommandInteraction } from 'discord.js';
```

Comment:
Should I move `CommandInteraction` to the start?

Reply:
They should be alphabetical, yes. Run prettier on your code and it'll handle that for you.